storybook/src/components/FileList/FileList.test.stories.tsx

```diff
@@ -19,7 +19,21 @@ export default meta
 
 type Story = StoryObj<typeof meta>
 
+const testFn = fn()
+
 export const Test: Story = {
-  render: (args) => renderComponentVariants(FileList, { args }),
+  args: {
+    children: [
+      <FileList.Item
+        file={new File(['sample1'], 'sample1.txt', { lastModified: Date.now(), type: 'text/plain' })}
+        key="1"
+        onDelete={() => testFn()}
+      />,
+    ],
+  },
+  play: async ({ canvas, userEvent }) => {
+    await userEvent.click(canvas.getByRole('button'))
+    await expect(testFn).toHaveBeenCalled()
```

Comment:
Same here, we test this [here](https://github.com/Amsterdam/design-system/blob/develop/packages/react/src/FileList/FileListItem.test.tsx#L55).